src/db/postgres/lrsql/postgres/sql/delete.sql

```diff
@@ -61,3 +61,51 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope::scope_enum;
+
+----------------------begin components of delete-actor-----
+-- :name delete-actor-st2st
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement 
```

Comment:
FYI in Postgres i believe you can make this ALL the same hug fn, just use semicolons and put them one line after another. It should work to inject the `:actor-ifi` into all of them

Reply:
Would that have parity with the SQLite version though? Don't we generally match up the query fns?

Reply:
no? I always saw it as we explicitly don't and that's the point of record.

Reply:
as these are clearly logically grouped w/the delete-actor prefix, I like Cliff's idea here